### PR TITLE
MyAccountScreen tests

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

https://github.com/tasitlabs/tasit/pull/315#discussion_r277054288

### Types of changes

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### Notes

Here is possible to verify that "too optimistic" behavior that I've said on Slack.
When an account is funded with ethers both mana and approval steps run in parallel updating the `creationActions` state object with `FUNDING_WITH_MANA` and `APPROVING_MARKETPLACE` making the `MyAccountScreen` shows the progress as 100%.

### CODE REVIEW

File: decentraland/screens/MyAccountScreen.test.js [hunk 1]
```diff
@@ -18,14 +22,46 @@ describe("MyAccountScreen", () => {
...
-                [AccountCreationStatus.FUNDING_WITH_ETH]: {},
+                [FUNDING_WITH_ETH]: {},
+              },
+            }}
+          />
+        )
+      ).toMatchSnapshot();
+    });
+
+    it("funding with ethers with null action (used when action is unknowed)", async () => {
```
Comment (pcowgill) [inline decentraland/screens/MyAccountScreen.test.js]:
```suggestion
    it("funding with ethers with null action (used when action is unknown)", async () => {
```

File: decentraland/screens/__snapshots__/MyAccountScreen.test.js.snap [hunk 2]
```diff
@@ -34,7 +34,75 @@ exports[`MyAccountScreen renders the component with account 1`] = `
...
+        "creationStatus": "FUNDING_WITH_ETH",
+        "name": "Funded with ETH",
+        "percentage": 0.25,
+        "status": "done",
+      },
+      Object {
+        "creationStatus": "FUNDING_WITH_MANA",
+        "name": "Funded with MANA tokens",
+        "percentage": 0.25,
+        "status": "done",
```
Comment (pcowgill) [inline decentraland/screens/__snapshots__/MyAccountScreen.test.js.snap]: Ah ok, so this isn't a bug though based on the way we've designed it. Both funding with mana and approving marketplace have been initiated, and until we store pending state, it's the best we can do

**Comment by pcowgill** [comment]
@marcelomorgado Tests failed since I renamed that test but didn't update the snapshot (and remove the one with the old name)

**Comment by marcelomorgado** [comment]
> @marcelomorgado Tests failed since I renamed that test but didn't update the snapshot (and remove the one with the old name)

Fixed.

**Pull request merged**